From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect dma-mapping.h include
Patch-mainline: never, kabi
References: kabi

Upstream commit b330104fa76df3eae6e199a23791fed5d35f06b4 (PCI: endpoint:
Use EPC's device in dma_alloc_coherent()/dma_free_coherent()) removed an
#include from pci-epc-core.c. It made the kABI checker to complain.

Just add the include back to make the kABI checker happy.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/pci/endpoint/pci-epc-core.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/pci/endpoint/pci-epc-core.c
+++ b/drivers/pci/endpoint/pci-epc-core.c
@@ -18,6 +18,7 @@
  */
 
 #include <linux/device.h>
+#include <linux/dma-mapping.h>
 #include <linux/slab.h>
 #include <linux/module.h>
 
